week-1-practise/level-2/3-currency-conversion.js

```diff
@@ -5,7 +5,9 @@
   Write a function that converts a price to USD (exchange rate is 1.4 $ to £)
 */
 
-function convertToUSD() {}
+function convertToUSD(price) {
+  return "$" + Math.round(price * 1.29631);
```

Comment:
Well done for putting in the effort to look up the current exchange rate and not just using the value in the question! I also like that you've tried to format it with a currency symbol, even though the question didn't ask for that.

I'm not sure that rounding is necessarily a good thing to do. We could be overcharging or undercharging our customers by removing the cents! How could you improve this?

As an extra (difficult) question, can you think of a time when adding the dollar sign would not be a good thing to do and might actually cause problems?